packages/api-client/src/conversation/ConversationAPI.ts

```diff
@@ -792,6 +794,43 @@ export class ConversationAPI {
     return response.data;
   }
 
+  /**
+   * Post an encrypted message to a conversation.
+   * @param messageData Mls message payload in TLS format. Please refer to the MLS specification for details.
+   * @see https://messaginglayersecurity.rocks/mls-protocol/draft-ietf-mls-protocol.html#name-message-framing
+   * @see https://staging-nginz-https.zinfra.io/swagger-ui/#!/conversations/postOtrMessage
+   */
+  public async postMlsMessage(messageData: Uint8Array): Promise<MlsEvent> {
+    const config: AxiosRequestConfig = {
+      data: {
+        description: messageData,
```

Comment:
i think theres something i dont fully understand but based on the swagger API the message body looks like a plaintext string?

Reply:
I was in doubt too but I did it this way cause I looked at the Model in swagger but think you're maybe right it makes more sense 
https://github.com/wireapp/wire-web-packages/pull/4292/commits/4d5200568eea097cefd3ef945431ba93abf55463